playbooks/installation/ldap_integration.adoc

```diff
@@ -20,259 +84,166 @@ toc::[]
-      surName: Whetsell
-      givenName: jamie
-      phone: 843-867-5309
-      ...  # based on what type of object they are using for the account
-  + dn: ou=Groups,dc=sample,dc=com
-    + dn: uid=admins,ou=Groups,dc=sample,dc=com
-      member: uid=jamie,ou=Users,dc=example,dc=com  # This is a pointer to the dn of a user.  This says the user is in the admin groups
-      member: uid=someotheruser,ou=Users,dc=example,dc=com
-    + dn: uid=jamie,ou=Users,dc=example,dc=com
-      member: uid=jamie,ou=Users,dc=example,dc=com # users can be members of multiple groups
-----
-      
-      
-== Useful LDAP Queries
+Another option is to add a filter to the query in which to further limit the users that are allowed to authenticate to OpenShift. To limit only user who are members of the _openshift_ group, the `(member=uid=admins,ou=Groups,dc=sample,dc=com)` can be applied which results in a _url_ represented as `url: "ldaps://ldap.example.com/ou=Users,dc=example,dc=com?uid?(member=uid=admins,ou=Groups,dc=sample,dc=com)`.
 
-Active Directory
-[source,ldap]
-----
-(&(objectClass=user)(sAMAccountName=yourUserName)
-  (memberof=CN=YourGroup,OU=Users,DC=YourDomain,DC=com))
-----
-  
-LDAP
-[source,ldap]
-----
-(&(objectClass=inetOrgPerson)(cn=uid)
-  (memberof=CN=YourGroup,OU=Users,DC=YourDomain,DC=com))
-----
+=== Manual Configuration 
 
-TIP: The member of overlay needs to be added to LDAP server for the member of query to work.
+To configure the `LDAPPasswordIdentityProvider` within OpenShift, modify the OpenShift master configuration file located at `/etc/orgin/master/master-config.yaml`
 
-ApacheDS pre 2.1
-----
-[source,ldap]
-(&(objectclass=groupOfNames)(cn=openshift)(member=cn=jamie,ou=People,dc=examplex,dc=com))
-----
+Locate the _identityProviders_ section of the OpenShift master configuration located at `/etc/origin/master/master-config.yaml` and configure the section with the value provided in the previous section.
 
-Example URL for master-config.yaml
-[source,ldap]
-----
-url: "ldap://10.3.9.248/ou=People,dc=opentlcx,dc=com?uid?sub?(&(objectClass=inetOrgPerson)(memberOf=cn=openshift,ou=Group,dc=opentlcx,dc=com))"
-----
+Once complete, restart the `atomic-openshift-master-api` (or `atomic-openshift-master` if running a single master) service to apply and enable the changes
 
-Example Ansible.cfg
-[source,yaml]
+[source,bash]
 ----
-openshift_master_identity_providers=[{'name': 'my_ldap_provider', 'challenge': 'true', 'login': 'true', 'kind': 'LDAPPasswordIdentityProvider', 'attributes': {'id': ['dn'], 'email': ['mail'], 'name': ['cn'], 'p\
-referredUsername': ['uid']}, 'bindDN': '', 'bindPassword': '', 'ca': '/path/to/my-ldap-cacert.crt', 'insecure': 'false', 'url': 'ldap://ldap.mycompany.com:389/ou=users,dc=mycompany,dc=com?uid'}]
+systemctl restart atomic-openshift-master-api
 ----
 
-== Adding a filter to a specific group
+IMPORTANT: These configurations must be applied on each master instance
+
+=== OpenShift Ansible Configuration
```

Comment:
I would like to make this section more prominent, and clearly recommended. Maybe we put this ahead of the manual method.